packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -81,9 +120,18 @@ const Example = () => (
 )
 ```
 
-We can use useErrorBoundary.setError in ErrorBoundaryFallback to make ErrorBoundary aware of the Error without throw.
+We can use useErrorBoundary.setError in ErrorBoundaryFallback to make `<ErrorBoundary/>` aware of the Error without throw.
```

Comment:
actually I usually use `setError` in children, not in ErrorBoundaryFallback😄

I think I'll use useErrorboundary.reset a lot in fallback component

Reply:
Oh! you're right, It's my mistake